.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -31,6 +31,7 @@ body:
       label: Version
       description: What version of Homarr are you running?
       options:
+        - 1.6.0
```

Comment:
This shouldn't be added in feature pull requests, because...
- we update this template when we release a new Homarr version
- when we need to roll back a certain commit, we accidentally also revert this change, which would not be wanted.
Therefore, please revert this change